Gems/GraphCanvas/Code/Source/Components/Slots/Data/DataSlotLayoutComponent.cpp

```diff
@@ -277,6 +277,14 @@ namespace GraphCanvas
         m_nodePropertyDisplay = aznew NodePropertyDisplayWidget();
         m_slotConnectionPin = aznew DataSlotConnectionPin(owner.GetEntityId());
         m_slotText = aznew GraphCanvasLabel();
+
+        if (const AZ::Entity* ownerEntity = owner.GetEntity())
+        {
+            if (const SlotComponent* slotComponent = ownerEntity->FindComponent<DataSlotComponent>())
+            {
+                m_isNameHidden = slotComponent->IsNameHidden();
+            }
+        }
```

Comment:
Is it possible to pass this down to the `DataSlotLayout` instead of having it query this from its owner?

Reply:
With the way GraphCanvas works, stringing m_isNameHidden to DataSlotLayout directly would be pretty awkward